serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc

```diff
@@ -1 +1,294 @@
-//= Mocking OpenAPI Services with Wiremock
\ No newline at end of file
+Mocking OpenAPI services with WireMock
+======================================
+// Metadata:
+:description: Mocking OpenAPI services with WireMock
+:keywords: kogito, workflow, quarkus, serverless, quarkus-cli, test, wiremock, openapi
+// links:
+:wiremock_url: https://wiremock.org/docs/
+:quarkus_test_resource_url: https://quarkus.io/guides/getting-started-testing#quarkus-test-resource
+// Referenced documentation pages.
+:getting-familiar-with-our-tooling: xref:getting-started/getting-familiar-with-our-tooling.adoc
+:create-your-first-workflow-service: xref:getting-started/create-your-first-workflow-service.adoc
+:orchestration-of-openapi-based-services: xref:service-orchestration/orchestration-of-openapi-based-services.adoc
+
+This document describes how to mock OpenAPI services using WireMock. The testing procedure in this document is based on the `serverless-workflow-examples/serverless-workflow-service-calls-quarkus` example application. You can access this example application in link:{kogito_examples_url}[Kogito Examples] GitHub repository.
```

Comment:
I think it's worth adding a link to the example directly rather than the main repo.